_maps/yogstation.json

```diff
@@ -1,5 +1,5 @@
 {
-    "map_name": "Yog Station",
+    "map_name": "YogStation",
```

Comment:
Oh and I removed this space because I can't have spaces in proc paths

Reply:
Please add a Travis test to ensure mapsnames don't have space in them